src/Console/ModelsCommand.php

```diff
@@ -456,7 +456,9 @@ protected function getPropertiesFromMethods($model)
                     //Use reflection to inspect the code, based on Illuminate/Support/SerializableClosure.php
                     $reflection = new \ReflectionMethod($model, $method);
                     // php 7.x type or fallback to docblock
-                    $type = (string) $reflection->getReturnType() ?: (string)$this->getReturnTypeFromDocBlock($reflection);
+                    $type = (string) $reflection->getReturnType()
+                        ? (string) $reflection->getReturnType()
+                        : (string)$this->getReturnTypeFromDocBlock($reflection);
```

Comment:
Still looks slightly wrong:
`? (string) $re…`
vs
`: (string)$thi`

Reply:
I saw Travis CI passed, but not good enough do you have a better way?